src/main/java/seedu/address/logic/commands/ExpenseCommand.java

```diff
@@ -41,8 +41,8 @@
             + PREFIX_EXPENSE + "10.0 "
             + PREFIX_DESCRIPTION + "Bubble tea";
 
-    public static final String MESSAGE_SUCCESS = "%d expense(s) successfully created as follows:\n%s\n";
-    public static final String MESSAGE_EXPENSE = "%s paid %s.\n  Description: %s\n";
+    public static final String MESSAGE_SUCCESS =
+            "Expense of %s by %s successfully created.\n  Description: %s\n  People involved: %s";
```

Comment:
Is the space for after the newline intentional?

Reply:
Intended for it to be a bit of an indentation

Reply:
Sounds good.